device-types/Datacom/DM4270-24XS-plus-2CX.yaml

```diff
@@ -0,0 +1,92 @@
+---
+manufacturer: Datacom
+model: DM4270 24XS+2CX
+slug: datacom-dm4270-24xs-plus-2cx
+part_number: 800.5231.xx
+u_height: 1.0
+is_full_depth: true
+airflow: side-to-rear
+comments: L2 L3 MPLS switch, wire speed, with 24 10 GE optical ports on SFP+ connectors and 3 optical ports 100 GE on QSFP28 connectors. Optionally the
+  uplink ports can be configured to operate as 4 40GE ports, disabling the 100GE ports. 1U metal enclosure for rack 19. Power supplies, MPLS license and
+  optical modules must be purchased separately
+weight: 3.15
+weight_unit: kg
+console-ports:
+  - name: Console0
+    type: rj-45
+  - name: Console1
+    type: usb-mini-b
+console-server-ports:
+  - name: Storage
+    type: usb-a
```

Comment:
Is this actually used as a console server port? Or is it just for the switch to access a USB storage device? If the latter, please remove this definition as it would be out of scope.

Reply:
removed